SwiftIsland/About/AboutViewController.swift

```diff
@@ -0,0 +1,57 @@
+//
+//  AboutViewController.swift
+//  SwiftIsland
+//
+//  Created by misteu on 01.10.19.
+//  Copyright © 2019 AppTrix AB. All rights reserved.
+//
+
+import UIKit
+
+class AboutViewController: UIViewController {
+  
+  private let dataManager = DataManager.shared
+  private var about: About? {
+    didSet {
+      updateLabels()
+    }
+  }
+  
+  @IBOutlet weak var appNameLabel: UILabel!
+  @IBOutlet weak var appVersionLabel: UILabel!
+  @IBOutlet weak var appDescriptionLabel: UILabel!
+  @IBOutlet weak var appOrganisersLabel: UILabel!
+  @IBOutlet weak var eventInfoLabel: UILabel!
+  @IBOutlet weak var githubLinkButton: UIButton!
+  @IBOutlet weak var contributorsLabel: UILabel!
+  
+  override func viewDidLoad() {
+    super.viewDidLoad()
+    loadDummyAboutData()
+  }
+  
+  private func loadDummyAboutData() {
+    
+    // TODO: remove and include cacheManager etc.
+    guard let dummyData = TestDataManager().getLocalTestAsset(forPath: "About_valid.json") else { return }
+    about = try? JSONDecoder().decode(About.self, from: dummyData)
+  }
```

Comment:
As your comment states, it should use the `CacheManager`, not use the `TestDataManager` (which should not be in the SwiftIsland target either)